spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/EnableModule.java

```diff
@@ -47,7 +46,7 @@
 @Configuration
 @MessageEndpoint
 @Import({RedisServiceConfiguration.class, RabbitServiceConfiguration.class,
-	ChannelBindingAdapterConfiguration.class, CodecConfiguration.class, ChannelBindingAdapterRunner.class,
```

Comment:
Now that Rabbit and Redis have `@Configuration` support, I think that we should move `RedisServiceConfiguration` as well as `RabbitServiceConfiguration` into the binder packages. There's no reason for them to be in SCS core.